components/brave_news/browser/resources/feed/Card.tsx

```diff
@@ -33,23 +33,35 @@ export const Title = styled.h3`
 
   &> a { all: unset; }
 `
-
-const HidableImage = ({ onError, ...rest }: React.DetailedHTMLProps<React.ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement>) => {
+type HidabeImageProps = React.DetailedHTMLProps<React.ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement> & {
+  imgUrl: string,
```

Comment:
Hey, we should still pass `src` here as the image soure - we can pull it out in the props destructure. As-is the props accept `imgUrl` and `src` (but src will be ignored)

Reply:
done